test/demo/createCenter.inttest.ts

```diff
@@ -34,7 +34,7 @@ describe('When creating a notification with the center showing', () => {
     });
 
     beforeEach(async () => {
-        testApp = await createApp(testManagerIdentity, {});
+        testApp = await createApp(testManagerIdentity, {url: 'http://localhost:3922/demo/app.html'});
```

Comment:
As this url is now used in lots of places, can we maybe put this in a `constants` file or something similar?

Reply:
Sure I can do that.  I tried to put it in the beforeEach or as a const in the files that needed it to keep it cleaner but if we think it would be better overall in one file that can be done

Reply:
There's already a `test/demo/utils/constants.ts` file that I think would be perfect for it.

Reply:
Ok buddy shes done.  That const really fit in the file like a glove.

Reply:
Truly a sight to behold. I'd be a fool not to approve this now.